extensions/amp-analytics/0.1/session-manager.js

```diff
@@ -121,20 +113,24 @@ export class SessionManager {
       !isSessionExpired(this.sessions_[type])
     ) {
       this.sessions_[type] = this.updateSession_(this.sessions_[type]);
+      if (opt_processing) {
+        opt_processing(this.sessions_[type]);
+      }
       this.setSession_(type, this.sessions_[type]);
       return Promise.resolve(this.sessions_[type]);
     }
 
-    return this.getOrCreateSession_(type);
+    return this.getOrCreateSession_(type, opt_processing);
   }
 
   /**
    * Get our session if it exists or creates it. Sets the session
    * in localStorage to update the access time.
    * @param {string} type
+   * @param {!Function} opt_processing
```

Comment:
```suggestion
   * @param {function=} opt_processing
```

Reply:
@rcebulko FYI `amp check-types` said that `function` is unrecognizable, apparently `Function` is the correct version.

Reply:
Updated suggestion. Should be `function()=`, doesn't matter too much at this stage though